### PR TITLE
Used Array.map's index instead of counting separately

### PR DESCRIPTION
**Pull request by NotWearingPants**

### CODE REVIEW

File: src/renderer/search-results-component.ts [hunk 1]
```diff
@@ -43,13 +43,8 @@ export const searchResultsComponent = Vue.extend({
...
         },
         update(searchResults: SearchResultItem[]) {
-            let counter = 0;
-
             const viewModel = searchResults.map(
-                (searchResult): SearchResultItemViewModel => {
-                    counter++;
-                    return createSearchResultItemViewModel(searchResult, counter);
-                },
+                (searchResult, counter): SearchResultItemViewModel => createSearchResultItemViewModel(searchResult, counter),
```
Comment (oliverschwendener) [inline src/renderer/search-results-component.ts:47]: In your implementation counter is `0` for the first item and before the count started at `1`.

**Review by oliverschwendener — CHANGES_REQUESTED** [review]
See my comment.

**Comment by oliverschwendener** [comment]
And please target the `dev` branch.

File: src/renderer/search-results-component.ts (hunk 1 above)
Comment (oliverschwendener) [reply src/renderer/search-results-component.ts:47]: Here is what I would suggest:

```typescript
const viewModel = searchResults.map(
    (searchResult, index): SearchResultItemViewModel =>
        createSearchResultItemViewModel(searchResult, index + 1),
);
```

Comment (NotWearingPants) [reply src/renderer/search-results-component.ts]: Whoops, fixed :)

**Review by oliverschwendener — APPROVED** [review]

**Pull request merged**